carcara/src/checker/mod.rs

```diff
@@ -555,6 +555,7 @@ impl<'c> ProofChecker<'c> {
             "la_mult_pos" => extras::la_mult_pos,
             "la_mult_neg" => extras::la_mult_neg,
             "mod_simplify" => extras::mod_simplify,
+            "bitblast_extract" => bvextract::extract,
```

Comment:
We should keep only the parser changes in this PR. The implementation of the actual rules should be in a different PR.

Reply:
ooops.